StudioCore/MapStudioNew.cs

```diff
@@ -1010,6 +1048,12 @@ private void Update(float deltaseconds)
             }
             ImGui.End();
 
+
+            //Global shortcut keys
+            if (InputTracker.GetControlShortcut(Key.S) && !_msbEditor.Viewport.ViewportSelected)
```

Comment:
What specifically does this fix?

Reply:
Previously, Ctrl+S hotkey only worked when the top left file menu was already open. This makes it so Ctrl+S works at all times (except for when viewport is being actively used, though the viewport check is pretty sloppy)